hummingbot/connector/derivative/hyperliquid_perpetual/hyperliquid_perpetual_utils.py

```
@@ -32,35 +32,50 @@ def validate_bool(value: str) -> Optional[str]:
 
 class HyperliquidPerpetualConfigMap(BaseConnectorConfigMap):
     connector: str = Field(default="hyperliquid_perpetual", client_data=None)
-    hyperliquid_perpetual_api_secret: SecretStr = Field(
-        default=...,
+    use_vault: bool = Field(
+        default="no",
         client_data=ClientFieldData(
-            prompt=lambda cm: "Enter your Arbitrum wallet private key",
-            is_secure=True,
+            prompt=lambda cm: "Do you want to use the vault address? (Set up at https://app.hyperliquid.xyz/vaults) (Yes/No)",
```

Ruff (0.14.14)

[warning] Unused lambda argument: `cm`

(ARG005)